src/index.js

```diff
@@ -5,6 +5,7 @@ const core = require('@actions/core');
 
 const URL = github.context.payload.pull_request.comments_url;
 const GITHUB_TOKEN = core.getInput("token") || process.env.token;
+const USE_MENTION = core.getInput("mention") || process.env.mention;
```

Comment:
Can you please rename this to `SHOULD_NOTIFY_USER` ?   I think the naming is clearer.